yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -561,45 +567,92 @@ func getAssessmentReportContentFromAnalyzeSchema() (err error) {
 		assessmentReport.SchemaSummary.DBObjects[i].InvalidCount = 0
 	}
 
-	unsupportedFeatures, err := fetchUnsupportedFeaturesOfSourceInYb(schemaAnalysisReport)
+	// fetching unsupportedFeaturing with the help of Issues report in SchemaReport
+	var unsupportedFeatures []UnsupportedFeature
+	var err error
+	switch source.DBType {
+	case ORACLE:
+		unsupportedFeatures, err = fetchUnsupportedOracleFeaturesFromSchemaReport(schemaAnalysisReport)
+	case POSTGRESQL:
+		unsupportedFeatures, err = fetchUnsupportedPGFeaturesFromSchemaReport(schemaAnalysisReport)
+	default:
+		panic(fmt.Sprintf("unsupported source db type %q", source.DBType))
+	}
 	if err != nil {
-		return fmt.Errorf("failed to fetch unsupported features: %w", err)
+		return fmt.Errorf("failed to fetch %s unsupported features: %w", source.DBType, err)
 	}
-	assessmentReport.UnsupportedFeatures = unsupportedFeatures
-
+	assessmentReport.UnsupportedFeatures = append(assessmentReport.UnsupportedFeatures, unsupportedFeatures...)
 	return nil
 }
 
-func fetchUnsupportedFeaturesOfSourceInYb(schemaAnalysisReport utils.SchemaReport) ([]UnsupportedFeature, error) {
-	switch source.DBType {
-	case POSTGRESQL:
-		return fetchUnsupportedFeaturesForPG(schemaAnalysisReport)
-	case ORACLE:
-		return nil, nil
-	default:
-		panic("unsupported source db type")
+var filterIssues = func(featureName, issueReason string, schemaAnalysisReport utils.SchemaReport, unsupportedFeatures *[]UnsupportedFeature) {
```

Comment:
You shouldn't have to pass a pointer to an array. You are only appending.
For slices, the actual array will be effectively passed by reference.